docs/pxe/dgxie.md

```diff
@@ -51,7 +51,7 @@ kubectl create configmap pxe-machines --from-file=config/machines.json
 Launch the DGXie service:
 
 ```sh
-helm install --values config/helm/dgxie.yml services/dgxie
+helm install --values config/helm/dgxie.yml k8s-services/dgxie
```

Comment:
Are there no slurm services? Should we not be doing /services/k8s like we are for other things?

Reply:
Ah, nevermind. I see you moved all the services into a examples directory and copy them over. That works well enough.